composer.json

```
@@ -67,6 +67,7 @@
     },
     "conflict": {
         "doctrine/doctrine-bundle": ">=3",
+        "psr/container": "1.1.0 || 2.0.0",
```

```suggestion
        "psr/container": "1.1.0",
```

`symfony/dependency-injection` already blocks the installation of 2.0, so there is no real need to add that conflict rule.